### ISSUE TITLE
Put a label for album image in the music file table for visual impaired accessibility

### ISSUE THREAD

**Issue by dreinn** [AzuraCast/AzuraCast#452]
When I am in "music files" section, the table is accessible, but please, before the title of the song there is a field that should be the album photo or similar.
This field should be labelled as "artwork" or "album image", because now I read:
 
a448bc140aa788c55d0279d3

this is only an example.
Reading these  strings for all the songs is quite frustrating!
I hope you understand.
Thank you!

**Comment by SlvrEagle23** [comment]
@dreinn Sorry for any difficulty that's causing you. I'll get it fixed immediately!

**Comment by SlvrEagle23** [comment]
@dreinn Should be fixed now. :)
